ner4opt/constants.py

```diff
@@ -0,0 +1,117 @@
+class Constants:
+    """Class for all constants."""
+
+    MODELS_DIRECTORY = 'models'
+
+    LEXICAL = 'lexical'
+    LEXICAL_PLUS = 'lexical_plus'
+    SEMANTIC = 'semantic'
+    HYBRID = 'hybrid'
+
```

Comment:
This comment is related design/naming comment in the main function of ner4opt.py. 

These constants give the impression that CRF is not used for deep models. 

How about; 


```
    # Lexical Classical Models 
    LEXICAL_MODEL_NAME = 'lexical.pkl'
    LEXICAL_PLUS_MODEL_NAME = 'lexical_plus.pkl'

    # Semantic Deep Models 
    # https://huggingface.co/skadio/ner4opt-roberta-v1
    # https://huggingface.co/skadio/ner4opt-roberta-v2 (used as part of hybrid)
    # The only difference between v1 and v2 is due inherent randomness during training
    # Refer to the training section for the details of training setup 
    SEMANTIC_MODEL_ROBERTA_V1 = 'skadio/ner4opt-roberta-v1'
    SEMANTIC_MODEL_ROBERTA_V2 = 'skadio/ner4opt-roberta-v2'

    # Hybrid Model using LEXICAL_PLUS_MODEL and SEMANTIC_MODEL_ROBERTA_V2 
    HYBRID_MODEL_NAME = 'hybrid.pkl'
```

Reply:
Slightly changed it as per your suggestions and also see my commentary above for the new changes.

Reply:
this is great, thank you!